Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -3485,6 +3497,22 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
   }
 }
 
+extension Syntax {
+  var isPostfixIfConfig: Bool {
```

Comment:
I'm a little wary of making this an extension on `Syntax`, since it makes it look like a natural API provided by swift-syntax (I know that's kind of the point of extensions, but in this case, the API is specialized enough that it doesn't feel like it should be given that kind of prominence).

Could you make this just a file-level private function that just takes the node as an argument, instead? Named something like `isNestedInPostfixIfConfig` (the current name reads like it's checking that the node *itself* is the postfix-if-config, rather than checking that it's a descendant of one).

Reply:
I pushed this change but haven't tested it yet -- I can't get `main` to build so I'll either need to figure that out or cherry-pick to the 5.6 branch. I hope to get to that later today.

Reply:
I just pulled it down and tested against `main`, and everything looks good, so I'll go ahead and merge and save you the trouble. Thanks again!